debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -1121,6 +1121,7 @@ public void testCustomPublicationNameUsed() throws Exception {
         waitForAvailableRecords(100, TimeUnit.MILLISECONDS);
 
         stopConnector(value -> assertThat(logInterceptor.containsMessage("Creating new publication 'cdc' for plugin 'PGOUTPUT'")).isTrue());
```

Comment:
Ah, this explains why the test was passing, as the log message already was correct. Go figure :)

Reply:
Exactly, now the test validates both :)